disnake/message.py

```diff
@@ -1370,6 +1372,9 @@ def _update(self, data: MessageUpdateEvent) -> None:
     def _handle_edited_timestamp(self, value: str) -> None:
         self._edited_timestamp = utils.parse_time(value)
 
+    def _handle_pinned_timestamp(self, value: str) -> None:
+        self._pinned_at = utils.parse_time(value)
+
```

Comment:
Since `pinned_at` won't be received in `MESSAGE_UPDATE`s, this method shouldn't be necessary. The pin iterator can set `_pinned_at` directly

Reply:
https://github.com/DisnakeDev/disnake/pull/1306/commits/a818a431bddaada682417816274f51ad7aba9a6d